### PR TITLE
Feature/add glue dq alerts

### PR DESCRIPTION
**Pull request by aryna-yukhnevich**
- if DQ rules triggered on the Glue table:

image

- If DQ rules triggered via Glue job:

image

- unit tests added

image

- code coverage = 91%

image

### CODE REVIEW

File: src/lib/event_mapper/glue_data_quality_event_mapper.py [hunk 1]
```diff
@@ -52,8 +74,8 @@ def get_message_body(self):
             super().create_table_row(["State", self.get_resource_state()], style)
         )
 
-        context = self.event.get("detail", {}).get("context", {})
-        if context.get("contextType") == "GLUE_DATA_CATALOG":
+        context, context_type, run_id = self._get_context_and_run_id()
+        if context_type == "GLUE_DATA_CATALOG":
```
Comment (natmasslove) [inline src/lib/event_mapper/glue_data_quality_event_mapper.py:78]: maybe we can use context type constants declared in glue_manager here?

File: src/lib/event_mapper/glue_data_quality_event_mapper.py [hunk 2]
```diff
@@ -6,29 +6,51 @@
...
-            else context.get("jobId")
-        )
+        context_type = context.get("contextType")
+
+        if context_type not in GlueManager.Data_Quality_Context_Types:
+            raise GlueDataQualityEventMapperException(
+                f"Unknown or missing context type in the DQ event: {self.event}"
+            )
+
+        run_id_key_map = {"GLUE_DATA_CATALOG": "runId", "GLUE_JOB": "jobId"}
```
Comment (natmasslove) [inline src/lib/event_mapper/glue_data_quality_event_mapper.py:43]: can we consider using Data_Quality_Context_Types constants here also

**Review by natmasslove — APPROVED** [review]
Looks good to me, thanks!
Just few minor comments (not critical)

File: src/lib/event_mapper/glue_data_quality_event_mapper.py (hunk 1 above)
Comment (aryna-yukhnevich) [reply src/lib/event_mapper/glue_data_quality_event_mapper.py:78]: sure, will update accordingly

**Pull request merged**